app/static/js/settings.js

```diff
@@ -1,23 +1,47 @@
-document.addEventListener('DOMContentLoaded', () => {
-    const tabBtn = document.getElementsByClassName("tab");
-    for (let i = 0; i < tabBtn.length; i++) {
-        const btn = document.getElementById("tab" + i);
-        btn.addEventListener('click', () => {
-            tabClick(btn.id, tabBtn);
-        });
-    }
-});
+document.addEventListener("DOMContentLoaded", () => {
+  const tabBtn = document.getElementsByClassName("tab");
+  for (let i = 0; i < tabBtn.length; i++) {
+    const btn = document.getElementById("tab" + i);
+    btn.addEventListener("click", () => {
+      tabClick(btn.id, tabBtn);
+    });
+  }
 
+  var menstrualSubscriptionSwitch = document.getElementById("switch3");
+  menstrualSubscriptionSwitch.addEventListener("click", () => {
+    btnState = menstrualSubscriptionSwitch.checked;
+    if (btnState) {
+    fetch('/menstrual-predictor/')
+    .then(response => {
 
-function tabClick(tab_id, tabBtn) {
-    let shownTab = document.querySelector(".tab-show");
-    let selectedTabContent = document.querySelector(`#${tab_id}-content`);
-    shownTab.classList.remove("tab-show");
-    shownTab.classList.add("tab-hide");
-    for (btn of tabBtn) {
-        btn.children[0].classList.remove("active");
+        text = response;
+        let subscriptionContainer = document.getElementById('menstrual-prediction-container');
+        subscriptionContainer.innerHTML = text;
+
+    })
+    //.then(body => {
+    //     console.log(body, 'body')
+    //     subscriptionContainer.innerHTML= body;
+    // });
+    // window.location = '/menstrual-predictor/'
+      console.log(menstrualSubscriptionSwitch.checked);
     }
-    document.getElementById(tab_id).classList.add("active");
-    selectedTabContent.classList.remove("tab-hide");
-    selectedTabContent.classList.add("tab-show");
+  });
+});
+async function loadSubscriptionPage(response){
+    data = await response.text();
+    return data;
+}
+function toggleMenstrualPredictor() {}
```

Comment:
?

Reply:
removed, it was supposed to be used once the settings are completed.